src/main/java/com/google/firebase/auth/internal/FirebaseTokenVerifier.java

```diff
@@ -101,60 +117,60 @@ public boolean verifyTokenAndSignature(IdToken token) throws FirebaseAuthExcepti
 
     if (header.getKeyId() == null) {
       if (isCustomToken) {
-        errorMessage = "verifyIdToken() expects an ID token, but was given a custom token.";
+        errorMessage = method + " expects " + articledShortName + ", but was given a custom token.";
```

Comment:
Can you be consistent on whether to use use concatenation or String.formatf? FWIW, String.formatf is my preference.

Reply:
Done